config/routes.rb

```diff
@@ -10,21 +10,32 @@
       confirmations: "active_admin/devise/confirmations",
       two_factor_authentication: "auth/two_factor_authentication",
     },
-    path_names: { sign_in: "sign-in", sign_out: "sign-out", two_factor_authentication: "two-factor-authentication" },
+    path_names: {
+      sign_in: "sign-in",
+      sign_out: "sign-out",
+      two_factor_authentication: "two-factor-authentication",
+    },
     sign_out_via: %i[get],
   }
 
   devise_scope :admin_user do
     get "admin/two-factor-authentication/resend", to: "auth/two_factor_authentication#show_resend"
   end
 
-  devise_for :users, controllers: {
-    passwords: "auth/passwords",
-    sessions: "auth/sessions",
-  }, path_names: { sign_in: "sign-in", sign_out: "sign-out" }
+  devise_for :users, {
+    path: :account,
+    controllers: {
+      passwords: "auth/passwords",
+      sessions: "auth/sessions",
+    },
+    path_names: {
+      sign_in: "sign-in",
+      sign_out: "sign-out",
+    },
+  }
 
   devise_scope :user do
-    get "confirmations/reset", to: "auth/passwords#reset_confirmation"
+    get "account/password/reset-confirmation", to: "auth/passwords#reset_confirmation"
```

Comment:
I think you can use `as: confirmations/reset` to update the URL without changing the named route helper so you don't need to change the controller code above. Think that's probably a little safer in case devise expects that named route to exist elsewhere.